### PR TITLE
[new release] ansi (0.5.0)

### PR DESCRIPTION
**Pull request by samoht**
ANSI escape sequence parser

- Project page: <a href="https://github.com/ocurrent/ansi">https://github.com/ocurrent/ansi</a>
- Documentation: <a href="https://ocurrent.github.io/ansi/">https://ocurrent.github.io/ansi/</a>

##### CHANGES:

* Rename to `ansi` (ocurrent/ansi#2, @samoht)
* ansi: 24-bit colour parsing (ocurrent/ansi#1, @copy)
* Split into its own repository.

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
The license field is missing. According to the LICENSE file it should be something like:
```
license: "Apache-2.0"
```

File: packages/ansi/ansi.0.5.0/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "ANSI escape sequence parser"
+description: """
+This package provides a basic ANSI escape parser,
+allowing the OCurrent web UI to show logs in colour."""
+maintainer: "talex5@gmail.com"
+authors: "talex5@gmail.com"
```
Comment (samoht) [inline packages/ansi/ansi.0.5.0/opam:7]:
```suggestion
authors: "talex5@gmail.com"
license: "Apache-2.0"
```

**Comment by samoht** [comment]
Also added in https://github.com/ocurrent/ansi/pull/4

**Comment by MisterDA** [comment]
Is it possible to add a deprecation/transition notice to the `current_ansi` package?

**Comment by samoht** [comment]
@MisterDA how do you want to do this? Via a post-install message?

**Comment by MisterDA** [comment]
Well, if there's no better way, yes ;)

**Pull request merged**